zuul/connection/github.py

```diff
@@ -185,6 +185,21 @@ def _event_pull_request_review(self, request):
         event.type = 'pr-review'
         return event
 
+    def _event_status(self, request):
+        body = request.json_body
+        action = body.get('action')
+        if action == 'pending':
+            return
+        pr_body = self.connection.getPullBySha(body['sha'])
+        if pr_body is None:
+            return
+
+        event = self._pull_request_to_event(pr_body)
+        event.account = self._get_sender(body)
+        event.comment = body.get('comment', {}).get('body')
```

Comment:
Do status events ever come with a comment? I don't see that in the [example payload](https://developer.github.com/v3/activity/events/types/#statusevent), and I'm not sure what this could be.